docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -0,0 +1,318 @@
+
+[source, xml]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-grpc</artifactId>
+</dependency>
+<dependency>
+ <groupId>com.google.code.findbugs</groupId>
+ <artifactId>jsr305</artifactId>
+ <scope>provided</scope>
+</dependency>
+----
+
+NOTE: The jsr305 is only used to compile the generated classes. It is not required at runtime.
+
+In the `<properties>` section, define the 2 following properties:
+
+[source, xml, subs="verbatim,attributes"]
+----
+<grpc.version>{grpc-version}</grpc.version>
+<protoc.version>{protoc-version}</protoc.version>
+----
+
+They configure the gRPC version and the `protoc` version.
+
+Finally, add to the `build` section the `os-maven-plugin` extension and the `protobuf-maven-plugin` configuration.
+
+[source, xml, subs="verbatim,attributes"]
+----
+<build>
+    <extensions>
+        <extension>
+            <groupId>kr.motd.maven</groupId>
+            <artifactId>os-maven-plugin</artifactId>
+            <version>{os-maven-plugin-version}</version>
+        </extension>
+    </extensions>
+
+    <plugins>
+        <plugin>
+            <groupId>org.xolstice.maven.plugins</groupId>
+            <artifactId>protobuf-maven-plugin</artifactId>
+            <version>{protobuf-maven-plugin-version}</version>
+            <configuration>
+                <protocArtifact>com.google.protobuf:protoc:${protoc.version}:exe:${os.detected.classifier}</protocArtifact>
+                <pluginId>grpc-java</pluginId>
+                <pluginArtifact>io.grpc:protoc-gen-grpc-java:${grpc.version}:exe:${os.detected.classifier}</pluginArtifact>
+                <protocPlugins>
+                    <protocPlugin>
+                        <id>quarkus-grpc-protoc-plugin</id>
+                        <groupId>io.quarkus</groupId>
+                        <artifactId>quarkus-grpc-protoc-plugin</artifactId>
+                        <version>999-SNAPSHOT</version>
+                        <mainClass>io.quarkus.grpc.protoc.plugin.MutinyGrpcGenerator</mainClass>
+                    </protocPlugin>
+                </protocPlugins>
+            </configuration>
+            <executions>
+                <execution>
+                    <id>compile</id>
+                    <goals>
+                        <goal>compile</goal>
+                        <goal>compile-custom</goal>
+                    </goals>
+                </execution>
+                <execution>
+                    <id>test-compile</id>
+                    <goals>
+                        <goal>test-compile</goal>
+                        <goal>test-compile-custom</goal>
+                    </goals>
+                </execution>
+            </executions>
+        </plugin>
+
+        <!-- ... -->
+    </plugins>
+</build>
+----
+
+The `protobuf-maven-plugin` that generates stub classes from your gRPC service definition (`proto` files).
+The class generation uses a tool named `protoc`, which is OS-specific.
```

Comment:
Looks like they should be callouts? At least that's what I suppose from how the sentences are worded?

Reply:
Fixed.